foam_controlwrapper/examples/foam_mesh_to_h5cuds.py

```diff
@@ -11,5 +11,4 @@
 path = '.'
 mesh_inside_wrapper = read_foammesh(name, path)
 mesh_file = H5CUDS.open("poiseuille.cuds")
```

Comment:
> "poiseuille.cuds" doe not exist anymore. 

oops.  I mean that the "poiseuille/" (openfoam file-directory) doe not exist anymore.   

note i did use the  old "poiseuille/" but it took a long time (i killed it after 10 minutes).  Is it possible to have a smaller example.